### DISCUSSION TITLE
Is there an easier way to access a variant of a color?

### DISCUSSION THREAD

**Discussion by kitze** [mantinedev/mantine#1980] [Q&A] [answered]
Right now I'm doing this:
```
 <Title
    sx={(theme) => ({
      color: theme.colors.purple[4],
    })}
>
   {title}
</Title>
```

But it would be way better/shorter if I can do `<Title color="purple.4">{title}</Title>`

Is this possible?

**Comment by rtivital** [comment, marked as answer]
No, currently we support only `sx` syntax

**Comment by josch0** [comment]
Thanks for adding this feature in 5.1.0!

May it be possible to also add a shortcut for the primary color?

`<Title color="primary">{title}</Title>`

and

`<Title color="primary.6">{title}</Title>`

Thanks,
Joachim

**Reply by rtivital** [reply]
Title component currently does not support `color` prop, I'll consider adding these features in next releases.

**Reply by josch0** [reply]
Ah sorry. Then "Title" was a bad example. I meant components in general wich support the `color` prop with the new indexed color feature.

**Comment by kitze** [comment]
@rtivital thanks so much for adding this.
